Dockerfile

```diff
@@ -2,6 +2,7 @@ ARG PHP_VERSION=8.0
 FROM php:${PHP_VERSION}-cli
 
 ENV COMPOSER_ALLOW_SUPERUSER 1
+ENV XDEBUG_MODE=coverage
```

Comment:
K cemu potrebujeme by-default zapnuty coverage mode? (trklo me to kdyz vidim, ze z AZ pipelines odebiras ten results step)

Reply:
infection testy to chcou